context/app/static/js/pages/Publication/style.js

```diff
@@ -4,6 +4,10 @@ import Paper from '@material-ui/core/Paper';
 const StyledPaper = styled(Paper)`
   padding: 20px 40px 20px 40px;
   margin-bottom: 16px;
+
+  h2 {
+    padding: 20px 0 20px 0;
+  }
```

Comment:
Sorry if jumping in here is wrong, but I prefer to style the component itself unless it's somehow inaccessible.

Reply:
The h2 here is raw html that we generate from the markdown and inject into react... so I think this is a case where it is inaccessible... I could add a comment to that effect?

(Jumping in is welcome! I might be misunderstanding something.)

Reply:
You could use shorthand `padding: 20px 0px;`, but not a big deal.